CS3230Project/View/ModifyDiagnosis.cs

```diff
@@ -101,8 +108,8 @@ private void SubmitChangesFooter1OnSubmitButtonEventHandler(object sender, Event
                         this.diagnosisDescriptionErrorMessage))
                 {
                     var diagnosis = new Diagnosis(this.diagnosis.DiagnosisId, this.diagnosis.AppointmentId,
-                        this.diagnosisDescriptionTextBox.Text, this.isFinalCheckBox.Checked,
-                        this.basedOnTestResultsCheckBox.Checked);
+                        this.diagnosisDescriptionTextBox.Text, (string)this.isFinalComboBox.SelectedItem == "True",
+                        (string)this.basedOnTestResultsComboBox.SelectedItem == "True");
```

Comment:
You reference the string "True" alot I would move that into a private field.

Reply:
Also, I know it is nitpicky but I would change the text all together to Yes or No, instead of true or false. That would be more intuitive and approachable to the user.

Reply:
Moved the string "True" to a private field.

Reply:
True / False has been changed to Yes / No